src/Core/Content/ImportExport/Service/FileService.php

```diff
@@ -48,6 +48,8 @@ public function storeFile(Context $context, \DateTimeInterface $expireDate, ?str
                 throw ImportExportException::fileNotReadable($sourcePath);
             }
             $this->filesystem->writeStream($path, $sourceStream);
+
+            fclose($sourceStream);
```

Comment:
you should check if it's still open. some weird filesystem adapters are closing the stream